src/api/spec/features/webui/requests_spec.rb

```diff
@@ -127,7 +127,7 @@
         find(:id, 'review_type').select('User')
         fill_in 'review_user', with: reviewer.login
         click_button 'Ok'
-        expect(page).to have_text("Open review for #{reviewer.login}")
+        expect(page).to have_text(/Open review for\s+#{reviewer.login}/)
```

Comment:
I would try to fix this in the code:

https://github.com/openSUSE/open-build-service/blob/master/src/api/app/views/webui/request/show.html.erb#L67

Maybe if you move these lines together here then there is no additional whitespace. This would also be a proper fix as this is more like a "workaround" I wold say.

Reply:
Then let's not invest too much time into it ...